configs/config.txt

```diff
@@ -61,4 +61,5 @@ dtparam=audio=on
 
 device_tree_address=0x02000000
 
-dtoverlay=vc4-fkms-v3d,cma-256
+# dtoverlay=vc4-fkms-v3d,cma-256
```

Comment:
I'm investigating this, it is related to video display drivers. I will test without deactivating it in the future, since you can stack overlays